CMSgov/onemac
# OY2-19588 - batchUpdateChangeRequestId implementation

Story: https://qmacbis.atlassian.net/browse/OY2-19588
Endpoint: https://d1git28twbsi47.cloudfront.net/

### Details

Create an admin function that can take a csv file and perform updates based on the given inputs.
Required data points: Old ID Format, New ID Format, Type, State

### Changes

- Created batchUpdateChangeRequestId admin function

### Implementation Notes

- I added a mapping in this function for Type given by business for "Medicaid Spa" -> "spa"; future stories for other types may want to add on new mappings

### Test Plan

1. Identify a medicaid spa change request to modify
2. Use the CSV format as found when exporting the given excel to CSV (Also see handlers/batchUpdateChangeRequest.json)
3. Open the test tab for the lambda in aws console and paste the csv content into the event json
4. Run
5. Verify in the log output which data successfully passed and which did not get updated
6. Verify in the database that updates were made
